website/content/docs/security/acl/tokens/create/create-a-service-token.mdx

```diff
@@ -0,0 +1,341 @@
+---
+layout: docs
+page_title: Create tokens for service registration
+description: >-
+  Learn how to create ACL tokens that your services can present to Consul servers so that they can register with the Consul catalog.
+---
+
+# Create a service token
+
+This topic describes how to create a token that you can use to discover services and to register services into the Consul catalog. If you are using Consul service mesh, this token can be used by a sidecar proxy to discover and route to other services.
```

Comment:
I think I fixed this wording:

> This topic describes how to create a token that you can use to register a service and discover services in the Consul catalog. If you are using Consul service mesh, this token can be used by a sidecar proxy to discover and route to other services.

I opted for "and discover services in the Catalog" because that is actually the permission it grants (i.e. it is not limited to just the dependencies of that service, although that is the purpose of those permissions)